custom_images/nginx/nginx.conf

```diff
@@ -20,6 +20,7 @@ http {
       proxy_buffer_size 64k;
       proxy_buffers 16 32k;
       proxy_busy_buffers_size 128k;
+      proxy_read_timeout 200s;
```

Comment:
Did we verify that this fixed their issue?

Reply:
Yep!